editors/code/src/installation/language_server.ts

```diff
@@ -100,15 +100,18 @@ export async function ensureLanguageServerBinary(
             try {
                 await downloadLatestLanguageServer(langServerSource);
             } catch (err) {
-                await vscode.window.showErrorMessage(
+                vscode.window.showErrorMessage(
                     `Failed to download language server from ${langServerSource.repo.name} ` +
                     `GitHub repository: ${err.message}`
                 );
 
-                await dns.resolve('www.google.com').catch(err => {
-                    console.error("DNS resolution failed, there might be an issue with Internet availability");
-                    console.error(err);
-                });
+                dns.resolve('www.google.com').then(
```

Comment:
Just a nit: Could we do someone more neutral than google?

Reply:
https://www.rust-lang.org ?

Reply:
Or http://example.com anyway...